lib/create-organization-api.ts

```diff
@@ -696,7 +696,9 @@ export default function createOrganizationApi(makeRequest: MakeRequest) {
      * })
      *
      * client.getOrganization('<org_id>')
-     * .then((org) => org.upsertAppDetails('app_definition_id', { value: 'tsren3s1....wn1e' }))
+     * .then((org) => org.upsertAppDetails('app_definition_id',
```

Comment:
It feels strange to me that this method is on org level and not part of the app definition api (https://github.com/contentful/contentful-management.js/blob/master/lib/create-app-definition-api.ts) but we already did the same for the signing secret 🤷